quirks/shared-credentials.json

```diff
@@ -268,6 +268,12 @@
             "postnl.be"
         ]
     },
+    {
+        "shared": [
+            "pretendo.network",
+            "pretendo.cc"
+            ]
```

Comment:
Can you shift the closing bracket back a few spaces/tab so it's aligned with line 272? You can look at the shared group above yours as an example.

Reply:
Resolved in commit https://github.com/apple/password-manager-resources/pull/632/commits/9b4b03c3f97ab70b98060c45e4a254a7ce9e4945